metrics/web/lib/dashboard/presentation/widgets/metrics_table_header.dart

```diff
@@ -11,13 +11,16 @@ class MetricsTableHeader extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final theme = MetricsTheme.of(context)
+    final metricsTableHeaderTheme = MetricsTheme.of(context)
         .projectMetricsTableTheme
         .metricsTableHeaderTheme;
+    final defaultFontFamily = Theme.of(context).textTheme.bodyText1.fontFamily;
+    final textStyle = metricsTableHeaderTheme.textStyle
+        .copyWith(fontFamily: defaultFontFamily);
```

Comment:
Wold it be better to merge the text style from the metrics theme with the default material text style instead of copying the font family? 
```suggestion
    final textStyle = DefaultTextStyle.of(context)
        .style
        .merge(metricsTableHeaderTheme.textStyle);
```